File: ruby/iniciante/1015.rb
p1 = gets.chomp.split(' ')
p2 = gets.chomp.split(' ')
x1 = p1[0].to_f
y1 = p1[1].to_f
x2 = p2[0].to_f
y2 = p2[1].to_f
puts '%.4f' % "#{Math.sqrt(((x2 - x1) ** 2)  + ((y2 - y1) ** 2))}"

File: python3/iniciante/1015.py
import math

def calcularDistancia(x1, y1, x2, y2):
    return math.sqrt((x2 - x1)**2 + (y2 - y1)**2)

x1, y1 = list(map(float, input().split()))
x2, y2 = list(map(float, input().split()))

distancia = calcularDistancia(x1, y1, x2, y2)
print("{:.4f}".format(distancia))
